cmsdb/processes/higgs.py

```diff
@@ -1422,26 +1422,40 @@
     name="ttzh",
     id=120000,
     label=r"$t\bar{t}ZH$",
+    xsecs={
+        # taken from https://cms.cern.ch/iCMS/jsp/db_notes/noteInfo.jsp?cmsnoteid=CMS%20AN-2022/122 (v25)
+        13: Number(1.535e-03, {
+            "scale": (0.019j, 0.068j),
+            "pdf": 0.030j,
+        }),
+    },
     aux={"production_mode_parent": ttvh},
 )
+# naive xsec scaling for 13.6 TeV using the ggF xsec
+ttzh.xsecs[13.6] = ttzh.xsecs[13] * h_ggf.xsecs[13.6] / h_ggf.xsecs[13]
 ttwh = ttvh.add_process(
     name="ttwh",
     id=130000,
     label=r"$t\bar{t}WH$",
+    xsecs={
+        13: Number(1.538e-03),
+    },
     aux={"production_mode_parent": ttvh},
 )
+# naive xsec scaling for 13.6 TeV using the ggF xsec
+ttwh.xsecs[13.6] = ttwh.xsecs[13] * h_ggf.xsecs[13.6] / h_ggf.xsecs[13]
```

Comment:
Could you move this into the `xsecs={}` definitions above? Maybe with a `:=` not to repeat the value for 13.